### PR TITLE
EID-Summary-Page

### PR DESCRIPTION
**Pull request by semujju**

### CODE REVIEW

File: omod/src/main/webapp/resources/css/ugandaemr-poc-styles.css [hunk 1]
```diff
@@ -9,11 +9,11 @@ font-weight: normal;
 }
 
 #99030{
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/css/ugandaemr-poc-styles.css]: @semujju What is this class for its not clear

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml [hunk 2]
```diff
@@ -0,0 +1,846 @@
+<htmlform formName="HMIS ACP - Exposed Infant Card Summary Form"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml]: @semujju add the EID Card number

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml [hunk 3]
```diff
@@ -0,0 +1,846 @@
...
+                                        </div>
+                                    </div>
+                                    <div class="card">
+                                        <div class="card-header">
+                                            Mother/Care Giver Follow Up Information
+                                        </div>
+                                        <div class="card-body">
+                                            <div class="row">
+                                                <div class="col-md-3">
+                                                    <label>Mother`s Name</label>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml]: @semujju this has a strange apostrophe character

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml [hunk 4]
```diff
@@ -0,0 +1,846 @@
...
+                                                <div class="col-md-4">
+                                                    <obs conceptId="162871" answerConceptIds="90003,90004"
+                                                         answerLabels="Yes,No" style="radio" labelText="Has this person been disclosed to?" class="horizontal"/>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="card">
+                                        <div class="card-header">
+                                            Mother`s History
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml]: @semujju Remove funny apostrophe

**Comment by ssmusoke** [comment]
@semujju 

1. Care Entry point 
  * Date Chart Opened must be greyed out since its read only 
  *  Date of NVP Start has strange text see 
Screenshot 2019-12-17 11 29 49

2. Date fields should all have the calendar icon 
3. Testing Information
  * Give each PCR its own card 
  * Result must only be enabled when a Test Date is selected 
  * 1st PCR date should be after date chart opened: currently it shows error cannot be in the future. Do this for all the tests 
  * Ensure validation that Date results given to care giver is after Test Date - was unable to test this 
 * Ensure that 2nd PCR date is after 1st PCR Date (if provided), 3rd PCR date is after 2nd PCR date (if provided), 18 month rapid test is after 3rd PCR date (if provided) 
  * Feeding method has mixed case text see 
Screenshot 2019-12-17 12 12 10
   * Final HIV status has 2 fields see 
Screenshot 2019-12-17 11 48 15

4. Linkage to care and Treatment followup
   * Referred to ART clinic - add better description to dates as they are not clear 
Screenshot 2019-12-17 12 16 01
   * Follow up information: add a separate card for each follow up attempt
   * Ensure that the date of the second follow up attempt is after the first, and the date of the 3rd follow up attempt is after the second

**Comment by semujju** [comment]
@ssmusoke 
1. Care enty point fields are fine locally as seen below.

Screenshot from 2019-12-18 14-21-57
Please verify if your JS is not echoing errors.

3. Testing Information
- Result must only be enabled when a Test Date is selected
This is fine locally as well as seen below.
Screenshot from 2019-12-18 14-27-44

I suspect you did not pull the latest code.

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml (hunk 3 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml]: This character has been removed and a correct one used

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml (hunk 4 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml]: Done

File: omod/src/main/webapp/resources/css/ugandaemr-poc-styles.css (hunk 1 above)
Comment (semujju) [reply omod/src/main/webapp/resources/css/ugandaemr-poc-styles.css]: This class had been removed

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml [hunk 5]
```diff
@@ -0,0 +1,907 @@
...
+                                            <div class="row">
+                                                <div class="col-md-4">
+                                                    <obs conceptId="162872" answerConceptIds="90003,90004,90001"
+                                                         answerLabels="Yes,No,Unknown" style="radio"
+                                                         labelText="Mother recieved ARVs for PMTCT" class="horizontal"/>
+                                                </div>
+                                                <div class="col-md-2 "></div>
+                                                <span class="container-padding"></span>
+                                                <label>Mother's ANC No.</label>
+                                                <div class="col-md-3">
```
Comment (solemabrothers) [inline omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml:515]: Align these two fields.

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml [hunk 6]
```diff
@@ -0,0 +1,907 @@
...
+                                                    <obs id="presumptive-refarral" conceptId="163003"
+                                                         answerConceptIds="1065,1066" style="radio" class="horizontal"/>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <h4>Follow Up Information</h4>
+                                    <div class="card">
+                                        <div class="card-header">
```
Comment (solemabrothers) [inline omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml:799]: Correct the positioning of the calendar icon for all date fields in this section

File: omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml (hunk 2 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/HMIS-ACP-015-ExposedInfantClinicalChart-SummaryPage.xml]: Done

**Pull request merged**